BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -220,6 +211,29 @@ open class GiniBankNetworkingScreenApiCoordinator: GiniScreenAPICoordinator, Gin
     public func didPressEnterManually() {
         self.resultsDelegate?.giniCaptureDidEnterManually()
     }
+
+    public func startSDK(withDocuments documents: [GiniCaptureDocument]?, animated: Bool = false) -> UIViewController {
+        configurationService?.fetchConfigurations(completion: { result in
+            switch result {
+            case .success(let configuration):
+                self.initializeAnalytics(with: configuration)
+            case .failure(let error): 
```

Comment:
[suggestion]: the error is not needed, we can remove it from here and satisfy swiftlint :D